#This is an inhouse patch to remove references to ccon which is a wrapper
#around console program that is not shipped in Solaris. An RFE has been 
#filed Bug 18544653 to port console command to Solaris. Once console is
#ported this patch becomes obsolete. 

--- App-ClusterSSH-4.02_01/bin/cssh.orig	2014-04-09 10:22:53.306791974 -0700
+++ App-ClusterSSH-4.02_01/bin/cssh	2014-04-09 10:30:00.989566222 -0700
@@ -16,14 +16,13 @@
 
 =head1 NAME
 
-cssh, crsh, ctel, ccon - Cluster administration tool
+cssh, crsh, ctel - Cluster administration tool
 
 =head1 SYNOPSIS
 
 S<< cssh [options] [[user@]<server>[:port]|<tag>] [...] >>
 S<< crsh [options] [[user@]<server>[:port]|<tag>] [...] >>
 S<< ctel [options] [<server>[:port]|<tag>] [...] >>
-S<< ccon [options] [[user@]<server>[:port]|<tag>] [...] >>
 
 =head1 DESCRIPTION
 
@@ -39,7 +38,7 @@
 Connections are opened via ssh so a correctly installed and configured
 ssh installation is required.  If, however, the program is called by "crsh"
 then the rsh protocol is used (and the communications channel is insecure),
-or by "ctel" then telnet is used, or by "ccon" then console is used.
+or by "ctel" then telnet is used.
 
 Extra caution should be taken when editing system files such as
 /etc/inet/hosts as lines may not necessarily be in the same order.  Assuming
@@ -88,12 +87,6 @@
 
 =item *
 
-If the code is called as ccon instead of cssh (i.e. a symlink called
-ccon points to the cssh file or the file is renamed) console is used as the
-communications protocol instead of ssh.
-
-=item *
-
 When using cssh on a large number of systems to connect back to a single
 system (e.g. you issue a command to the cluster to scp a file from a given
 location) and when these connections require authentication (i.e. you are
@@ -334,10 +327,6 @@
 
 S<$ crsh server1 server2>
 
-=item Use console with master as the primary server instead of ssh
-
-S<$ ccon -M master server1 server2>
-
 =back
 
 =head1 FILES
